### PR TITLE
1.1.3

### PR DESCRIPTION
**Pull request by kt3k**
Release PR for 1.1.3

ref: 1.1.2 https://github.com/denoland/fresh/pull/831

@lucacasonato

### CODE REVIEW

File: docs/concepts/updating.md [hunk 1]
```diff
@@ -51,14 +51,14 @@ You will be prompted to confirm the changes that will be made to your project.
...
 
 ```diff
   {
     "imports": {
 -     "$fresh/": "https://deno.land/x/fresh@1.0.2/",
-+     "$fresh/": "https://deno.land/x/fresh@1.1.2/",
++     "$fresh/": "https://deno.land/x/fresh@1.1.3/",
 
 -     "preact": "https://esm.sh/preact@10.8.1",
 -     "preact/": "https://esm.sh/preact@10.8.1/",
```
Comment (ry) [inline docs/concepts/updating.md:64]: preact not necessary?

**Review by ry — APPROVED** [review]
LGTM

File: docs/concepts/updating.md (hunk 1 above)
Comment (bartlomieju) [reply docs/concepts/updating.md:64]: This diff looks fishy. Is it a bug in GH that shows both sides as removed?

Screenshot 2023-02-04 at 03 11 33

Comment (kt3k) [reply docs/concepts/updating.md:64]: This is not pull request diff, but `diff` syntax highlight in markdown code block. (These lines are not changed, but just highlighted red)

Comment (kt3k) [reply docs/concepts/updating.md:64]: スクリーンショット 2023-02-04 21 22 36

These lines are folded below

**Comment by arisris** [comment]
hmmm. to many showcase addition 😞

**Comment by rojvv** [comment]
FINALLY

**Pull request merged**